@stellar/design-system/src/components/CopyText/index.tsx

```diff
@@ -1,24 +1,44 @@
 import React, { useState } from "react";
 import CopyToClipboard from "react-copy-to-clipboard";
 import { Tooltip } from "../Tooltip";
-import { FloaterPlacement } from "../Floater";
 import "./styles.scss";
 
-interface CopyTextProps {
+/** */
+export interface CopyTextProps {
+  /** Text to copy */
   textToCopy: string;
+  /** Label/text to display when copy action is done @defaultValue Copied */
   doneLabel?: string;
-  tooltipPlacement?: FloaterPlacement;
+  /** Position of the tooltip @defaultValue `bottom` */
+  tooltipPlacement?:
+    | "top"
+    | "right"
+    | "bottom"
+    | "left"
+    | "top-start"
+    | "top-end"
+    | "right-start"
+    | "right-end"
+    | "bottom-start"
+    | "bottom-end"
+    | "left-start"
+    | "left-end";
+  /** Title text to show on hover @defaultValue Copy */
   title?: string;
-  children: React.ReactElement | string;
+  /** Copy click element */
+  children: React.ReactElement;
 }
 
+/**
+ * Use `CopyText` component to copy a string. Done action label can be displayed in a tooltip, by default it will replace component’s label inline. We’re using [react-copy-to-clipboard](https://github.com/nkbt/react-copy-to-clipboard) to handle the copy part.
```

Comment:
This is the text that shows up in the tooltip when the copy action is done. By default, we show "Copied", but it can be customized using `doneLabel` property.

![image](https://github.com/stellar/stellar-design-system/assets/7346473/89dc0740-12f3-485f-8a0f-da3b663d40ad)